lib/std/array_list.zig

```diff
@@ -79,19 +92,21 @@ pub fn ArrayListAligned(comptime T: type, comptime alignment: ?u29) type {
             };
         }
 
+        /// Creates a copy of this list, but with no memory managment.
```

Comment:
```suggestion
        /// Initializes an ArrayListUnmanaged with the `items` and `capacity` fields from this ArrayList.
```

"copy of this list" could be misleading